src/property-accessors.coffee

```diff
@@ -1,5 +1,5 @@
 Mixin = require 'mixto'
-WeakMap = global.WeakMap ? require('harmony-collections').WeakMap
+WeakMap = require 'es6-weak-map'
```

Comment:
@Termina1 Was there a reason this couldn't still use `global.WeakMap` directly when available?